src/DssSpell.sol

```diff
@@ -126,198 +137,513 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
+    uint256 internal constant ONE_PCT_RATE      = 1000000000315522921573372069;
+    uint256 internal constant TWO_FIVE_PCT_RATE = 1000000000782997609082909351;
+    uint256 internal constant FOUR_PCT_RATE     = 1000000001243680656318820312;
+
+    // --- MATH ---
+    uint256 internal constant MILLION           = 10 ** 6;
+    uint256 internal constant WAD               = 10 ** 18;
+    uint256 internal constant RAY               = 10 ** 27;
```

Comment:
```suggestion
    uint256 internal constant MILLION = 10 ** 6;
    uint256 internal constant WAD     = 10 ** 18;
    uint256 internal constant RAY     = 10 ** 27;
```

Reply:
Resolving as this appears to have been pulled in.

Reply:
i was trying to remove the spaces because i thought we padded per block so these should be aligned to `MILLION` not the `*RATE`s above

Reply:
if we want to keep this, feel free to resolve again, but this is different than goerli

Reply:
done